### PR TITLE
Update docs about `hasura-cluster replicate -j`

### PR DESCRIPTION
**Pull request by johnclary**
👀 📖 👍 

---
#### Ship list
- [ ] Code reviewed 
- [ ] Product manager approved
- [ ] Added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#) if applicable

### CODE REVIEW

File: moped-database/README.md [hunk 1]
```diff
@@ -94,16 +94,16 @@ Configure your environment:
...
 ```bash
 $ ./hasura-cluster replicate
 ```
 
-If you would prefer to bypass downloading a new copy of production and instead just use whatever the last snapshot in the folder, add `--use-latest-snapshot` or `-j` 
+If you would prefer to bypass downloading a new copy of production and instead just use whatever the last snapshot in the folder, add `--use-any-snapshot` or `-j` 
 
 ```bash
-$ ./hasura-cluster --use-latest-snapshot
+$ ./hasura-cluster --use-any-snapshot
```
Comment (mddilley) [inline moped-database/README.md:106]: @johnclary thanks for updating all of this 🙏 can you update this line to include the replicate part of the command?

Comment (johnclary) [reply moped-database/README.md]: yes—thanks!

**Review by mddilley — APPROVED** [review]
🚢 📚🚢

**Review by chiaberry — APPROVED** [review]
😬 👍🏼

**Pull request merged**